src/Integrations/Greenhouse/Greenhouse.php

```diff
@@ -74,23 +84,35 @@ public function getForm(array $formAdditionalProps): string
 			return '';
 		}
 
+		// Return form to the frontend.
+		return $this->buildForm(
+			$this->getFormFields((string) $formId),
+			$formAdditionalProps
+		);
+	}
+
+	/**
+	 * Get Greenhouse maped form fields.
+	 *
+	 * @param string $formId Form Id.
+	 *
+	 * @return array<int, array<string, mixed>>
+	 */
+	public function getFormFields(string $formId): array
+	{
 		// Get Job Id.
 		$jobId = $this->getSettingsValue(SettingsGreenhouse::SETTINGS_GREENHOUSE_JOB_ID_KEY, (string) $formId);
 		if (empty($jobId)) {
-			return '';
+			return [];
 		}
 
 		// Get Job questions.
 		$questions = $this->greenhouseClient->getJobQuestions($jobId);
 		if (empty($questions)) {
-			return '';
+			return [];
 		}
 
-		// Return form to the frontend.
-		return $this->buildForm(
-			$this->getFields($questions, (string) $formId),
-			$formAdditionalProps
-		);
+		return $this->getFields($questions, (string) $formId);
```

Comment:
Is the `$formId` some numerical value or some string identifier? Like `form-12345`? Or just `123456`?

Reply:
as I see it is numerical only but I cant be sure so in whole project I used string for all formId identifiers.